python-server.py

```diff
@@ -40,6 +41,7 @@ def do_GET(self):
             self.send_error(404, 'File Not Found: %s' % self.path)
 
 if __name__ == '__main__':
-    server = ThreadedHTTPServer(('0.0.0.0', 80), MainHandler)
+    port = int(sys.argv[1]) if len(sys.argv) > 1 else 80
```

Comment:
It seems too implicit to pick up the first argument. Can you update it to pick up on an explicit flag only?